portal/src/app/components/home/components/experiment/components/table-options/components/select-columns-popup/utils/convert-data-to-options.utils.test.ts

```diff
@@ -17,6 +17,14 @@ describe('convertDataToOptions', () => {
       {
         label: 'results.averageMemory',
         value: TableOptionsData.options[2]
+      },
+      {
+        label: 'results.bytes_throughput',
+        value: 'Throughput (bytes/sec)'
```

Comment:
use `TableOptionsData.options[3]` and `TableOptionsData.options[4]`, just for the arrangement of the code